packages/react-component-library/src/components/PhaseBanner/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Badge from '../Badge'
+
+const PhaseBanner = ({ phase, html }) => (
+  <div className={`rn-phase-banner rn-phase-banner--${phase}`}>
+    <div className="container">
+      <Badge size="small" state="primary" type="solid">
+        {phase}
+      </Badge>
+      <span
+        className="rn-phase-banner__text"
+        dangerouslySetInnerHTML={{ __html: html }}
```

Comment:
dangerous by name..... This is why I'd rather use children, you can still default it.